msg/DistanceSensorModeChangeRequest.msg

```diff
@@ -0,0 +1,5 @@
+uint64 timestamp		# time since system start (microseconds)
+
+uint8 req_mode 			# requested mode of operation
```

Comment:
"req" sounds a lot like "required" - how about being more specific with "request_on_off"? To make it clear that it's only about on/off, not other obscure "mode"